content/docs/legacy-context.md

```diff
@@ -6,7 +6,7 @@ permalink: docs/legacy-context.html
 
 > Note:
 >
-> The legacy context API will be removed in version 17.
+> The legacy context API will be removed in a future major version.
```

Comment:
Could we move the entire "legacy context" section its own document link to it here? I think that would reduce the potential for confusion.

Reply:
Isn't this the legacy context document?

Reply:
Oh lol, my bad :D